terraform/api_gateway.tf.json.template.py

```diff
@@ -132,6 +135,20 @@ def for_domain(cls, domain):
     'status': '$context.status'
 }
 
+
+def add_priority(rule: MutableJSON) -> MutableJSON:
+    global rule_priority
```

Comment:
No mutable global variables, please. Note where I added suggested adding the call, which you deviated from.